test/core/client_channel/lb_policy/xds_override_host_test.cc

```diff
@@ -123,15 +112,16 @@ TEST_F(XdsOverrideHostTest, SwapChildPolicy) {
     ASSERT_TRUE(subchannel->ConnectionRequested());
     subchannel->SetConnectivityState(GRPC_CHANNEL_CONNECTING);
     subchannel->SetConnectivityState(GRPC_CHANNEL_READY);
-    EXPECT_NE(ExpectState(GRPC_CHANNEL_READY), nullptr);
+    picker = ExpectState(GRPC_CHANNEL_READY);
+    EXPECT_NE(picker, nullptr);
+    ExpectPickComplete(picker.get());
```

Comment:
We should check that the picker is handing out the expected address.

Reply:
This would basically be testing `round_robin`. I believe it is inefficient to duplicate those tests.
Note that there's a check all addresses are picked a few lines below.